types/networks/src/lib.rs

```diff
@@ -1,6 +1,5 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
-
 #[macro_use]
 extern crate lazy_static;
```

Comment:
Can this be removed?

Reply:
Unfortunately not

Reply:
Why not?

Reply:
Oh, we're using `lazy_static!` in drand.rs.

Reply:
I think I would prefer to import the macro explicitly but I guess that can wait until another PR.